src/main/java/mocacong/server/service/CafeService.java

```diff
@@ -323,6 +327,16 @@ private void validateCafeImagesCounts(List<MultipartFile> cafeImages) {
         }
     }
 
+    private void validateOwnedCafeImagesCounts(Cafe cafe, Member member, List<MultipartFile> cafeImages) {
+        List<CafeImage> ownedCafeImages = cafe.getCafeImages().stream()
+                .filter(cafeImage -> cafeImage.isOwned(member))
+                .collect(Collectors.toList());
+
+        if (ownedCafeImages.size() + cafeImages.size() > CAFE_IMAGES_PER_MEMBER_LIMIT_COUNTS) {
+            throw new ExceedCageImagesTotalCountsException();
+        }
```

Comment:
아래와 같이 변수명을 조금 더 descriptive 하게 작성해주어도 좋을 듯한데 어떤가요? 

`ownedCafeImages` -> `currentOwnedCafeImages`
`cafeImages` -> `requestCafeImages`

취향에 따라 갈릴 수 있어보이는 코멘트이므로 한번 확인만 해주세요 ㅎㅎ

```suggestion
        List<CafeImage> currentOwnedCafeImages = cafe.getCafeImages()
                .stream()
                .filter(cafeImage -> cafeImage.isOwned(member))
                .collect(Collectors.toList());

        if (currentOwnedCafeImages.size() + requestCafeImages.size() > CAFE_IMAGES_PER_MEMBER_LIMIT_COUNTS) {
            throw new ExceedCageImagesTotalCountsException();
        }
```